sass/pages/_assets.scss

```diff
@@ -18,29 +18,109 @@
         margin-bottom: 52px;
     }
 
-    .card {
-        margin: 0;
-        height: 8rem;
-    }
+    .asset-card {
+        display: grid;
+        grid-template-areas:
+            "banner"
+            "desc"
+            "tags";
+        grid-template-rows: 12rem 5.5rem 2rem;
 
-    .card-title {
-        font-size: 1.3rem;
-        overflow-wrap: break-word;
-    }
+        text-decoration: none;
+        overflow: hidden;
+        border-radius: $border-radius;
+        background-color: $syntax-theme-background;
+        color: $color-white;
+        border: 2px solid #383838;
 
-    .card-description {
-        font-size: 1rem;
-        height: initial;
-    }
+        &:hover {
+            background-color: $card-hover-background;
+            border: 2px solid #6b6b6b;
+        }
 
-    .card-image > img {
-        display: block;
-        height: auto;
-        max-width: 100%;
-    }
+        &__banner {
+            grid-area: banner;
+            display: block;
+            position: relative;
 
-    a {
-        text-decoration: none;
-        color: $link-color;
+            background: linear-gradient(45deg,
+                    $default-image-background-color 0%,
+                    $card-hover-background 40%,
+                    $default-image-background-color 100%);
+
+            img {
+                width: 100%;
+                height: 100%;
+                object-fit: contain;
+
+                background-color: $default-image-background-color;
+            }
+        }
+
+        &__title {
+            position: absolute;
+            width: 100%;
+            bottom: 0;
+
+            padding: 0.6rem;
```

Comment:
This shrinks the vertical padding for the title, this way it doesn't look so "airy"… and I stop. 😇 
Anyway, if you think `0.6rem` is too much please reduce as you wish… it's just that I find `0.3rem` too tight.

```suggestion
            padding: 0.3rem 0.6rem;
```

![image](https://user-images.githubusercontent.com/188612/179687507-c93c7833-0f51-4e93-8b5a-30d5f699e02d.png)

Reply:
😍 I love it, I that was the missing part.